### DISCUSSION TITLE
What error of this?

### DISCUSSION THREAD

**Discussion by Ryangithub10** [wbthomason/packer.nvim#1122] [General]
i have installed packer, but when i configurate it, it will be:
![error](https://user-images.githubusercontent.com/86828950/200556702-8eeca1ac-8cf2-4b29-a3ad-96443d9a7572.PNG)

I was configurate it correctly and i was read a _Quick start_ on the README.
So, please the help.

Btw i'm using nvim on _windows_, so i'm install packer on windows too, And i have to migrating my config file from the previous  _vimscript_ to _Lua_ and i new with it.

_Sorry for my bad grammar_

**Comment by bryant-the-coder** [comment]
`:source %`

**Comment by yutkat** [comment]
Show me your init.lua.

**Comment by Ryangithub10** [comment]
No no no, it's done now! I'm just put my plugin config into the bottom.

Sorry for this, my problem was solve now before you solve it 😅.

But now, want to know how to specify plugin's destination when i'm was install it?

_Sorry for bad grammar again 😅_
